js/mediator/apiService.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+class ApiService {
+  constructor() {
+    this.arrDirector = [];
+  }
+
+  sortResult() {
+    const arr = this.arrDirector.sort((a, b) => b.value - a.value);
+    const btnCenter = new MurkupBtnAuthor(
+      arr,
+      ".content__horizontal-btn",
+      "center"
+    );
+    const btnRight = new MurkupBtnAuthor(
+      arr,
+      ".content__vertical-btn",
+      "right"
+    );
+    btnCenter.createBtn();
+    btnRight.createBtn();
+  }
+
+  returnId(value) {
```

Comment:
This method is returning array of items id, so it could be named as `returnIds`, don't you think?